### PR TITLE
Fix edge startup logs formatting

### PR DESCRIPTION
**Pull request by varunpuranik**
Fix the Edge startup logs formatting to be like the logs via serilog, so that they can be parsed correctly.

### CODE REVIEW

File: edge-agent/docker/windows/amd64/Dockerfile [hunk 1]
```diff
@@ -11,6 +11,4 @@ WORKDIR /app
 
 COPY $EXE_DIR/ ./
 
-ENV IPInterfaceName vEthernet (Ethernet)
```
Comment (varunpuranik) [inline edge-agent/docker/windows/amd64/Dockerfile]: Unrelated to this PR, but unused.

File: edge-agent/docker/windows/arm32v7/Dockerfile [hunk 2]
```diff
@@ -8,6 +8,4 @@ WORKDIR /app
 
 COPY $EXE_DIR/ ./
 
-ENV IPInterfaceName vEthernet (Ethernet)
```
Comment (varunpuranik) [inline edge-agent/docker/windows/arm32v7/Dockerfile]: Unrelated to this PR, but unused.

File: edge-hub/docker/linux/arm32v7/Dockerfile [hunk 3]
```diff
@@ -15,5 +15,5 @@ EXPOSE 443/tcp
 USER edgehubuser
 ENV OptimizeForPerformance false
 ENV MqttEventsProcessorThreadCount 1
-CMD echo "$(date --utc +"[%Y-%m-%d %H:%M:%S %:z]"): Starting Edge Hub" && \
+CMD echo "$(date --utc +"%Y-%m-%d %H:%M:%S %:z") Starting Edge Hub" && \
```
Comment (philipktlin) [inline edge-hub/docker/linux/arm32v7/Dockerfile]: nice to have: inject this time format by ARG.  May be just created a pbi to follow up later.

**Comment by aribeironovaes** [comment]
Looks good. 

I've created some Dockerfile that is checked in already, since you are here, can you fix them as well? I'm creating a bunch of them today, I will try to follow this pattern. Once you are done ping me and I can sign off again.

File: edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs [hunk 4]
```diff
@@ -24,7 +25,7 @@ public class Program
 
         public static int Main()
         {
-            Console.WriteLine($"[{DateTime.UtcNow.ToString("MM/dd/yyyy hh:mm:ss.fff tt")}] Edge Agent Main()");
+            Console.WriteLine($"{DateTime.UtcNow.ToString("yyyy-MM-dd HH:mm:ss.fff zzz", CultureInfo.InvariantCulture)} Edge Agent Main()");
```
Comment (philipktlin) [inline edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs]: Nice to have: suggest to make time format as a constant to share across multiple projects.

File: edge-hub/docker/linux/arm32v7/Dockerfile (hunk 3 above)
Comment (varunpuranik) [reply edge-hub/docker/linux/arm32v7/Dockerfile]: Seems a bit of an overkill.. what would be the scenario?

File: edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs (hunk 4 above)
Comment (varunpuranik) [reply edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs]: Yeah, would have been nice but it is a little tricky to use in the logger template itself. 
So added an method on DateTime itself that can be reused

File: edge-hub/docker/linux/arm32v7/Dockerfile (hunk 3 above)
Comment (philipktlin) [reply edge-hub/docker/linux/arm32v7/Dockerfile]: nice-to-have, just for code optimization.

**Pull request merged**